uw-frame-components/portal/timeout/services.js

```diff
@@ -11,8 +11,9 @@ define(['angular', 'jquery'], function(angular, $) {
     * getTimeout() - Gets the shib session, but returns just the timeout
                      and the time of that timeout.
     **/
-    .factory('PortalShibbolethService', ['$http', 'miscService', 'SERVICE_LOC',
-        function($http, miscService, SERVICE_LOC) {
+    .factory('portalShibbolethService', ['$http', '$log',
```

Comment:
:question: Does this service put a hard dependency on running uPortal with Shibboleth?